content/docs/getting-started/hello-world.md

```diff
@@ -33,9 +33,9 @@ channels:
           type: string
           pattern: '^hello .+$'</code></pre>
 
-The first line tells us this is an AsyncAPI document and it's using version 2.0.0 of the specification. This line doesn't have to be the first one but it's a recommended practice.
+The first line of the specification starts with the document type (AsyncAPI) and the version (2.0.0) as a recommended practice.
 
-The second line is an identifier for our application. It is mandatory and must be unique so, when doing real stuff, don't use `hello-world-app` but instead something more "unique" like `urn:com:mycompany:hello-world-app`.
+The second line identifies the application and is both required and unique. In a real environment using 'urn:com:mycompany:hello-world-app' is considered unique and preferred rather than 'hello-world-app', for example.
```

Comment:
```suggestion
The second line identifies the application and is both required and unique. In a real environment using 'urn:com:mycompany:hello-world-app' is preferred rather than 'hello-world-app', for example.
```

Reply:
@lifewingmate what do you think about this better? ☝️